deployment/docker-compose/template.backend.env

```diff
@@ -16,3 +16,4 @@ SES_SENDER_EMAIL=no-reply@example.com
 FRONTEND_URL=http://localhost:3000
 AWS_ACCESS_KEY_ID=aws_access_key
 AWS_SECRET_ACCESS_KEY=aws_secret_key
+AWS_REGION=ap-south-1
```

Comment:
There is no need of it as this file already have `SES_REGION` on just couples of lines before. If you are getting the NoRegion error then it means that `SES_REGION` does not have value in it.